controllers/handlers_kustomize.go

```diff
@@ -898,7 +899,8 @@ func extractTarGz(src, dest string) error {
 func instantiateResourceWithSubstituteValues(templateName string, resource []byte,
 	substituteValues map[string]string, logger logr.Logger) ([]byte, error) {
 
-	tmpl, err := template.New(templateName).Option("missingkey=error").Funcs(funcmap.SveltosFuncMap()).Parse(string(resource))
+	tmpl, err := template.New(templateName).Option("missingkey=error").Funcs(funcmap.SveltosFuncMap()).
+		Funcs(sprig.TxtFuncMap()).Parse(string(resource))
```

Comment:
Can you give me an example of what this will allow which is not today? 

In any case the addition should go inside the SveltosFuncMap so we add to single place.

Reply:
I'm sorry, I'll create a new FeatureRequest here. 🙏 
https://github.com/projectsveltos/sveltos/issues/480

Reply:
Thank you! 😭 
I'll add it! 👍

Reply:
https://github.com/projectsveltos/libsveltos/pull/433
👍